src/template.html

```diff
@@ -11,20 +11,20 @@
 <body>
     <main>
         <section class="score-section">
-            <header>
+            <div class="header">
                 <h1 class="logo">Leaderboard</h1>
                 <div class="header-container">
                     <div class="title-container">
-                        <img  src="check.png" alt="check">
+                        <img src="check.png" alt="check">
                         <h1>Recent scores</h1>
```

Comment:
- [OPTIONAL] The image you put here is not loading properly and renders a brown image in the UI. I would advise you to fix this so that it doesn't affect the outlook of the UI. 👍